bin/npmDownload.js

```diff
@@ -10,15 +11,16 @@ let program = require('commander'),
 program
     .version('0.1.4')
     .option('-p, --packages [packages]', 'A list of space separated [packages].', packages)
+    .option('-j, --packagesFromJsonFile [jsonFilePath]', 'path to package.json file.', packagesFromJsonFile)
     .option('-d, --dependencies', 'Download dependencies.')
     .option('-e, --devDependencies', 'Download dev dependencies.')
     .option('-a, --allVersions', 'Download all versions of each package.')
     .option('-o, --output [directory]', 'The output [directory].')
     .option('-7, --zipIt', '7Zips the downloaded files.')
     .parse(process.argv);
 
-if (!program.packages) {
-    winston.error("No npmPackage were given!");
+if (!program.packages && !program.packagesFromJsonFile) {
```

Comment:
no. if nither of them appear in the command, then it's an error.
you can call: -j "[path-to-json-file]" or: -p ...